contentctl/api.py

```diff
@@ -0,0 +1,124 @@
+from pathlib import Path
+from typing import Any, Union, Type
+#from contentctl import contentctl
+from contentctl.input.yml_reader import YmlReader
+from contentctl.objects.config import test_common, test_servers, test
+
+
+
+def configFromFile(path:Path=Path("contentctl.yml"), config: dict[str,Any]={}, 
```

Comment:
nit: not something to be addressed in this MR since I know it extends beyond this file, but we should consider going with more traditional PEP8 Python style guidelines and use snake-case for function definitions

Reply:
We discussed improving our style guidelines/requirements last week.
During reviews, let's start "requiring" that new functions/classes follow guidelines.
Fixing existing ones to follow those guidelines is a bit more involved, but there's no good reason not to at least start enforcing it for new things :)

Reply:
Agreed! Yeah I think if we're onboard to do PEP8 as a style guide, then doing it as we go forward is a good idea :)